packages/xo-server-netbox/src/index.js

```diff
@@ -341,6 +342,32 @@ class Netbox {
         netboxVm.platform = platform.id
       }
 
+      const netboxVmTags = []
+      for (const tag of vm.tags) {
+        const slug = slugify(tag)
+        let netboxTag = find(netboxTags, { slug })
+        if (netboxTag === undefined) {
+          // TODO: Should we also delete/update tags in Netbox?
```

Comment:
Yes, at least partially, because it's weird if tags are added in Netbox when added to the real VM, but not removed when removed from the real one.

Reply:
I was thinking about that but I'm not sure it's a good idea since, as always, you may delete some manual data like the tag's description or even its color. For instance, you might have a "Maintenance" tag with a custom description in Netbox and you want to keep it even when none of your VMs are in maintenance. Or maybe it could be an option?

But even if we decide to delete unused tags, I don't think there's any way to know which ones have been created by XO.

Reply:
I may have misunderstood, tags are not deleted/updated but they are correctly unlinked from the VM objects right?

In that case that seems perfect.